opensourceleg/hardware/actuators.py

```diff
@@ -687,6 +705,24 @@ def set_motor_position(self, position: float) -> None:
             ),
         )
 
+    def set_max_case_temperature(self, temperature: float) -> None:
```

Comment:
Is there a reason to do this versus the property getter/setter syntax? 
```
@max_case_temperature.setter
def max_case_temperature(self, temperature: float) -> None:
"""
        Set the maximum temperature of the motor.
        Args:
            temperature (float): temperature in degrees Celsius
        """
        self._max_case_temperature = temperature
```

Reply:
Most elements of the library follow the `set_variable(value)` routine instead of the `variable = value` or setter/getter way and IMO calling a method to set value to a critical variable is a better way to handle this instead of letting the user use assignment statements.

Reply:
Got it, sounds good!